code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -106,6 +106,6 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 	layer = FLY_LAYER
 	appearance_flags = TILE_BOUND
 
-/obj/effect/overlay/gas/New(state)
+/obj/effect/overlay/gas/Initialize(mapload, state)
 	. = ..()
 	icon_state = state
```

Comment:
This is used in a global var initialization to generate a cache and should not be be changed to initialize since it runs during world init before the mc even starts initializing

Reply:
Nope. I really want you to treat this as a datum, not an atom.